articles/api/management/v2/tokens-flows.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Describes what changed in the flow for generating Auth0 Management APIv2 tokens and why.
+section: apis
+---
+
+# Flows for generating Auth0 Management APIv2 Tokens
+
+Recently we changed the process to get a Management APIv2 Token. This article explains what changed, why this was done and how you can work around it (not recommended).
+
+## What changed and why
+
+### The User Experience
+
+Until recently you could generate a Management APIv2 Token directly from the Management API explorer. You selected the scopes, according to the endpoint you wanted to invoke, and got a token from that same page.
+
+That way was very easy but it was also __very insecure__. So we changed it.
+
+The new way uses the [OAuth 2.0 Client Credentials grant](/api-auth/grant/client-credentials). This new flow requires some initial configuration: [set up a Non Interactive Client](#1-create-a-client) and [authorize it to access the Management APIv2](#2-authorize-the-client).
+
+Once this is done you can get a new token either [using the dashboard](#4-get-the-token) (if you use the API sporadically) or [by configuring a server process](#1-get-a-token) (if you need a token frequently) that will get a new token every 24 hours.
+
+<div class="alert alert-info">For details on how to follow this new process refer to <a href="/api/management/v2/tokens">The Auth0 Management APIv2 Token</a></div>
+
+#### Why this changed
+
+In order to generate the token, the Management API had access to your __Global Client Secret__ (used to sign the token). This is information that should __not__ be exposed to web browsers.
+
+The new OAuth 2.0 Client Credentials grant implementation does not pose such risks, since the secret is not exposed on your browser. Once you do the initial configuration, you can get a token either by visiting the dashboard, or by making a simple `POST` request to `/oauth/token`.
+
+However, with regards to the manual process, we do understand that changing screens is not always the best user experience, so we are looking into ways to make the new flow more intuitive.
+
+
+### The Validity Period
+
+With the previous flow the tokens never expired. With the new flow all Management APIv2 Tokens __expire after 24 hours__.
+
+#### Why this changed
+
+Having a token that never expires can be very risky, in case an attacher gets hold of it. If the token expires within a few hours the attacker has only a small window to access your protected resources.
+
+
+## Can I still get a non-expiring token?
+
+Using Auth0 UI, no you cannot. However, if you need a work-around to the old way, you can generate a token using [JWT.io](https://jwt.io/).
+
+<div class="alert alert-danger">Long-lived tokens compromise your security. Following this process is <strong>NOT</strong> recommended.</div>
```

Comment:
It's also worth mentioning that you cannot revoke them, something like "Long-lived and non-revocable tokens"